var/spack/repos/builtin/packages/snakemake/package.py

```diff
@@ -24,6 +26,12 @@ class Snakemake(PythonPackage):
 
     depends_on('python@3.3:3.6', when='@:5')
```

Comment:
Both of these should presumably be `type=('build', 'run')`? Unless it also links or Python isn't needed at run-time.

Reply:
Yes, you're right. Fixed it.